src/lib/types/currency/balance.ts

```diff
@@ -3,10 +3,10 @@ import type { Option } from "lib/types";
 export interface Balance {
   amount: string;
   id: string;
-  name: Option<string>;
+  name?: Option<string>;
```

Comment:
why both ? and Option

Reply:
name and symbol field wont be included in Unsupported token case (those only present in Celatone API)